src/lib.rs

```diff
@@ -1049,14 +1053,14 @@ where
     /// What key is next among the active set
     fn peek_active_next(&self) -> Option<&<S as SortKey<T>>::Key> {
         let n = self.active_queue.peek_min();
-        n.map(|v| v.current_key())
+        n.map(ShardIter::current_key)
```

Comment:
Stuff like this matters in terms of the amount of code that gets shoved at the linker.

Reply:
I mean, does this really? I would be shocked if the compiler doesn't inline this single method call. And this is arguably more readable.

Reply:
I'm sure it does inline it.  The question would be at what step in the compilation process.  At the very least it needs to emit debug information to include the lambda in call stacks.

Reply:
But also this is an automated clippy fix.